### PR TITLE
Navigation: Add react-native-screens

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: RNAndroidPlayground/rnkiwimobile/build.gradle [hunk 1]
```diff
@@ -48,6 +48,7 @@ dependencies {
     api "com.trinerdis.skypicker:react-native-maps:$reactNativeMaps-SNAPSHOT"
     api "com.trinerdis.skypicker:react-native-vector-icons:$reactNativeVectorIcons-SNAPSHOT"
     api "com.trinerdis.skypicker:react-native-tooltips:$reactNativeTooltips-SNAPSHOT"
+    api "com.trinerdis.skypicker:react-native-screens:$reactNativeTooltips-SNAPSHOT"
```
Comment (ferrannp) [inline RNAndroidPlayground/rnkiwimobile/build.gradle]: Should be $reactNativeScreens.

Comment (tbergq) [reply RNAndroidPlayground/rnkiwimobile/build.gradle]: Should be fixed now ☺️

**Comment by ferrannp** [comment]
Let's wait for #1124 to be merged.

**Comment by tbergq** [comment]
we should wait for #1124 before merging this PR

**Pull request closed without merging**